recipe/meta.yaml

```diff
@@ -23,6 +22,12 @@ requirements:
     - python
 
 test:
+  imports:
+    - xlrd
+  requires:
+    - pip
+  commands:
+    - pip check
   files:
     - test.xls
 
```

Comment:
Could you please remove quote `'` and double quote `"` signs in `home`, `license`, and `summary`? They are redundant because they are already strings.
And remove `doc_source_url` because `doc_url` is already exists

Reply:
Thanks, still struggling with these constantly non-conformant about section